Shared/UI/Face Detection/FaceImageCaptureViewController.swift

```diff
@@ -13,57 +13,84 @@ import UIKit
 import MetalKit
 import CoreImage.CIFilterBuiltins
 
-class FaceDetectionViewController: ImageCaptureViewController {
+/// A view controller that allows a user to capture an image of their face.
+/// A live preview of the camera is shown on the main view.
+class FaceImageCaptureViewController: ViewController {
```

Comment:
I refactored the image capture view controllers.

What was once "ImageCaptureViewController" is now "PhotoCaptureSession". It's not a VC at all, but rather just a wrapper around all the session setup. This way it can be used in a variety of view controllers.